index.js

```diff
@@ -21,33 +21,46 @@ const port = process.env.PORT;
 async function createApp() {
   await connectToDb();
 
+  app.use(
+    expressWinston.logger({
+      transports: [new winston.transports.Console()],
+      format: winston.format.combine(
+        winston.format.colorize(),
+        winston.format.json()
+      ),
+    })
+  );
+
   const app = express();
   // Middleware: parse the body to json
   app.use(bodyParser.json());
   // Api documentation
   app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerDocument));
```

Comment:
To-do: Create a Winston logger and add it to the res object, so it can be used later to log